src/app/components/challenge/challengesubmit/challengesubmit.component.html

```diff
@@ -27,125 +27,258 @@
+              code-btn"
+              (click)="copyTextToClipboard('pip install ')">
+              <i class="fa fa-clipboard" aria-hidden="true" alt="Copy to
+                clipboard."></i>
+            </button>
+          </li>
+          <li class="content">Add your EvalAI account token to evalai-cli<br>
+            <code>$ evalai set_token {{authToken}}</code>
+            <button class="btn ev-btn-dark btn-waves-effect waves-dark grad-btn
+              code-btn"
+              (click)="copyTextToClipboard('evalai set_token ')">
+              <i class="fa fa-clipboard" aria-hidden="true" alt="Copy to
+                clipboard."></i>
+            </button>
+          </li>
+          <li class="content">Push docker image to EvalAI docker registry<br>
+            <code>$ evalai push &lt;image&gt;:&lt;tag&gt; --phase
+              &lt;phase_name&gt;</code>
+            <button class="btn ev-btn-dark btn-waves-effect waves-dark grad-btn
+              code-btn"
+              (click)="copyTextToClipboard('evalai push <image>:<tag> --phase
+                <phase_name>')">
+                  <i class="fa fa-clipboard" aria-hidden="true" alt="Copy to
+                    clipboard."></i>
+                </button>
+              </li>
+              <li class="content">For more commands, please refer to <a
+                  class="text-highlight"
+                  href="https://evalai-cli.cloudcv.org/" target="_blank">evalai-cli
+                  documentation.</a>
+              </li>
+            </ol>
+          </div>
+        </div>
+      </div>
 
-            <div class="form">
-              <div class="row row-lr-margin">
-                <div class="col-md-10 col-sm-10 col-lr-pad submit-upload-file">
-                  <app-input (change)="validateInput($event.target.value)"
-                    [label]="'input_file'" [accept]="'.json, .zip, .txt, .tsv, .gz'"
-                    [placeholder]="''" [type]="'file'" [value]="''"
-                    #formsubmit></app-input>
+      <div *ngIf="challenge.is_docker_based" class="ev-md-container
+        ev-card-panel">
+        <span class="fs-20 fw-light">
+          Submission limits
+        </span><br><br>
+        <span class="fs-15">
+          <strong class="fw-semibold content">Team name:</strong>
+          {{phaseRemainingSubmissions.participant_team}}
+        </span>
+        <table align="center" class="centered responsive-table">
+          <thead>
+            <tr>
+              <th class="content">Phase details</th>
+              <th class="content">Remaining submissions</th>
+              <th class="content">Submission command</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr *ngFor="let key of phaseRemainingSubmissions.phases">
+              <td class="content">{{key.name}}
+                <br>
+                <strong class="text-med-black">Start date: </strong>{{key.start_date
```

Comment:
Here, also. It would be good if you could open this in the editor and take care of all these cases.

Reply:
yup, done